libraries/ms-common/inc/usart.h

```diff
@@ -0,0 +1,123 @@
+#pragma once
+// USART Interface
+#include <stdint.h>
+
+#include "status.h"
+#include "interrupt.h"
+#include "gpio.h"
+
+// For setting word length
+typedef enum {
+	USART_WORD_LENGTH_8b = 0,
```

Comment:
The [linter](https://travis-ci.org/uw-midsun/firmware/jobs/200740277) seems to indicate that you're using tabs. Our style guide suggests 2 space indents.

You can run ``make lint`` to make sure it passes all the linting rules from your local machine (or Vagrant) so you don't have to wait for the Travis CI test to kick in